src/PageView/Timer/Timer.js

```diff
@@ -96,48 +108,47 @@ class Timer extends React.Component {
         }
     }
 
-    countDown = (secs) => {
-        let seconds = this.state.seconds - 1;
-        this.setState({
-            time: this.secondsToTime(seconds),
-            seconds: seconds,
-        });
-        if (seconds === 0) {
-            clearInterval(this.timer);
-        }
-    }
-
+    
     render() {
         return (
             <div className="Timer">
                 <div className="grid-container">
-                    <div className="digit-timer">
-                        <span className="digit-hour">
+                    <div className="titles">
+                        <span className="active" >Timer   </span >
+                        <span >
+                            <Link to="/stopwatch">
+                                Stopwatch
+                            </Link>
+                        </span >
+                    </div>
+
+                    <div className="display-timer">
+                        <span className="display-hour">
```

Comment:
You don't necessarily need three different spans for seconds, min and hours. you can combine all of them inside one span.